tests/input/resources.yaml

```diff
@@ -138,6 +63,28 @@ simulation_resources:
         source_uri: "file://path/to/file"
       - filename: the_name_of_second_file.json
         source_uri: "file://path/to/second/file"
+    input:
```

Comment:
I think that we should document all three inputs to Abaqus here: AluminiumMaterialCard, ConcreteMaterialCard and AbaqusConfiguration.

Reply:
These are just dummy examples.

Reply:
Good point